wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Commands.java

```diff
@@ -25,6 +25,16 @@ public static Command none() {
     return new InstantCommand();
   }
 
+  /**
+   * Constructs a command that does nothing until interrupted.
+   *
+   * @param requirements subsystems the action requires
+   * @return the command
+   */
+  public static Command idle(Subsystem... requirements) {
+    return new RunCommand(() -> {}, requirements);
```

Comment:
Delegate to the `run` factory rather than calling RunCommand directly. (Applies to C++ as well)